articles/article-116-esoteric-programming-languages.md

```diff
@@ -0,0 +1,27 @@
+<!-- BEGIN TITLE -->
+# Esoteric Science
+<!-- END TITLE -->
+
+<!-- BEGIN BODY -->
+There are thousands of programming languages. But some of them are just laborious and inefficient. They are not designed for practical usage but to implement different aspects of unusual language concepts. Those programming languages are called esoteric. One of them is called Brainfuck. It consists of the eight language commands:
+
+<  > + -  . , [ ]
+
+There are more comments needed to explain the code than the size of the code itself.
+It is possible to write any program with this language the question solely is if one wants to.
```

Comment:
Language suggestion: It is possible to write any program with this language. But the question is, if one really wants.